src/components/include/utils/lock.h

```diff
@@ -82,7 +102,11 @@ class SpinMutex {
   ~SpinMutex() {}
 
  private:
+#ifdef QT_PORT
+  QAtomicInteger<unsigned int> state_;
+#else
   volatile unsigned int state_;
```

Comment:
because this variables not changed in QT branches

________________________________

This e-mail and any attachment(s) are intended only for the recipient(s) named above and others who have been specifically authorized to receive them. They may contain confidential information. If you are not the intended recipient, please do not read this email or its attachment(s). Furthermore, you are hereby notified that any dissemination, distribution or copying of this e-mail and any attachment(s) is strictly prohibited. If you have received this e-mail in error, please immediately notify the sender by replying to this e-mail and then delete this e-mail and any attachment(s) or copies thereof from your system. Thank you.